core/src/env_platform/db_handler.bal

```diff
@@ -62,9 +62,9 @@ function saveApplication(TreeRemovalForm form) returns boolean|error {
     mongodb:DatabaseError? inserted = applicationCollection->insert(application);
 
     if (inserted is mongodb:DatabaseError) {
-        log:printDebug("An error occurred while saving the application with ID: " + application.applicationId + ". " + inserted.reason() + ".") ;
+        log:printDebug("An error occurred while saving the application with ID: " + application.applicationId.toString() + ". " + inserted.reason().toString() + ".");
```

Comment:
Isn't the reason in string format?

Reply:
Yes.

Reply:
Then why are we calling the `toString()` there?

Reply:
I will check it.

Reply:
Yes I think we can remove that

Reply:
<img width="1433" alt="Screenshot 2020-05-16 at 08 07 07" src="https://user-images.githubusercontent.com/32588172/82108546-c6646b80-974c-11ea-8624-8d218af51347.png">

Reply:
Yes @yashodgayashan is right, we have to toString the reason, otherwise it throws an error. This is because of the complications when doing string concatenations in Ballerina I believe.

Reply:
@sanjulamadurapperuma Yes. Then we have to go with toString()